src/main/cpp/commands/arm/IncrementArmExtendCommand.cpp

```diff
@@ -28,6 +28,9 @@ void IncrementArmExtendCommand::Execute() {
 }
 
 bool IncrementArmExtendCommand::IsFinished() {
-    // Always end this command.
-    return true;
-}
\ No newline at end of file
+    bool at = false;
+
+    OKC_CALL(arm_->AtExtendSetpoint(&at));
+
+    return at;
```

Comment:
Is there any reason we can't return true and just handle the state in the subsystem

Reply:
I guess for incrementing it doesn't make sense, but the point was so that arm commands were blocking in auto